lib/tool-bar-view.js

```diff
@@ -47,19 +45,14 @@ export default class ToolBarView {
         } else {
           this.hide();
         }
+      }),
+      atom.config.onDidChange('tool-bar.fullWidth', ({newValue, oldValue}) => {
```

Comment:
Yeah, I moved it into the giant `this.subscriptions.add` above it. It's just hard to see unless you expand the entire file.